README.yaml

```diff
@@ -5,40 +5,54 @@
 #
 
 # Name of this project
-name: terraform-example-module
+name: terraform-bioanalyze
 
 # Logo for this project
 #logo: docs/logo.png
 
 # License of this project
-license: "APACHE2"
+#license: "APACHE2"
 
 # Copyrights
-copyrights:
-  - name: "Cloud Posse, LLC"
-    url: "https://cloudposse.com"
-    year: "2020"
+# copyrights:
+#   - name: "Dabble of DevOps, SCorp"
+#     url: "https://www.dabbleofdevops.com"
+#     year: "2020"
 
-# Canonical GitHub repo
-github_repo: cloudposse/terraform-example-module
+# # Canonical GitHub repo
+# github_repo: dabble-of-devops-biodeploy/terraform-aws-eks-jupyterhub
```

Comment:
`# github_repo: dabble-of-devops-bioanalyze/terraform-aws-bioanalyze-onboarding`

Reply:
I think the auto format is failing because some of the config variables are commented out.